stable/metricbeat/templates/clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "metricbeat.fullname" . }}
+  labels:
+    app: {{ template "metricbeat.name" . }}
+    chart: {{ template "metricbeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups: [""]
+  resources:
+  - namespaces
+  - pods
+  verbs: ["get", "list", "watch"]
```

Comment:
starting with 6.4, Metricbeat monitors more entities and requires more permissions to query them, this should look more like: https://github.com/elastic/beats/blob/master/deploy/kubernetes/metricbeat/metricbeat-role.yaml

Reply:
It would be very nice advice, thanks!
I modified clusterrole.yaml based on the settings shown in that url.